pkgs/top-level/all-packages.nix

```diff
@@ -36244,4 +36244,6 @@ with pkgs;
   swift-corelibs-libdispatch = callPackage ../development/libraries/swift-corelibs-libdispatch { };
 
   swaysettings = callPackage ../applications/misc/swaysettings { };
+
+  showmethekey = callPackage ../applications/video/showmethekey { };
```

Comment:
Please (roughly) alphabetically sort this, don't toss it at the bottom of the file.